llvm/utils/TableGen/CallingConvEmitter.cpp

```diff
@@ -125,6 +133,21 @@ void CallingConvEmitter::emitCallingConv(const Record *CC, raw_ostream &O) {
   O << "}\n";
 }
 
+// Return the name of the specified Init (DefInit or StringInit), with a
+// namespace qualifier if the corresponding record contains one.
+std::string CallingConvEmitter::getQualifiedNameFromInit(const Init *I) {
```

Comment:
nit: call this getRegisterName or getQualifiedRegisterName or something like that to indicate that its not generic now and deals with registers specially?

Reply:
Done.